.github/workflows/build.yml

```diff
@@ -0,0 +1,64 @@
+name: Build
+
+on:
+  workflow_call:
+
+env:
+  FMU_FILE_NAME: OSMPCameraSensor.fmu #TODO: Adjust to file name of your FMU
```

Comment:
@ClemensLinnhoff this is outdated now, as we use the repo name as FMU name, right?

Reply:
Yes, but let's merge this PR first, then adapt to naming the FMU according to the repo. This requirement did not exist, when @MarzenaFranek startet, so I would like to keep it separate.